docs/medical-api/api-reference/network/list-networks.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: "List Networks"
```

Comment:
We're not really listing the networks (those would be CommonWell, Carequality, etc.), but the entries on those networks' directories. One could argue we're listing gateways and thus the endpoint should be `/gateway`, but we used `/network` to keep the concept that we're listing network entries - and we don't plan to list the networks themselves.

Once we align on this, all new doc pages need to match that definition.

Reply:
Great, in this case wouldn't we also want to name the route /network-entry or something similar? I've been kind of pattern matching as far as naming goes simply because i can't tell what a good name is without building intuition on the domain.

Reply:
Yeah I agree, now that I know more I feel like in general the /network route is a little confusing - they're entries in a registry. in this case even the concept of the network is a bit of an implementation detail, as they're really like "MedicalDataProviders" or something. Though maybe NetworkEntries is a solid middleground.

LMK what you think and ill go rename everything in this PR.